hw/ip/spi_host/data/spi_host_testplan.hjson

```diff
@@ -325,12 +340,39 @@
             Collect coverage that all events was enabled and seen
             '''
     }
+    {
+      name: interrupt_cg
+      desc: '''
+            - check that we see all scenarios that can cause an interrupt and that the interrupts is fired.
+            - also check we can clear the interrupt
+            '''
+    }
     {
       name: different_ch_settings_cg
       desc: '''
-            - check that multiple end points was running at different speeds concurrently.
-            - collect coverage that multiple SPI endpoints was run with different config opts
+            - Check that multiple end points was running at different speeds concurrently.
+            - Collect coverage that multiple SPI endpoints was run with different config opts
+            '''
+    }
+    {
+      name: stall_cg
+      desc: '''
+            - Check that the DUT stalls if a command is configured to transmit data but the tx fifo is empty
+            - check that the data is transmitted when written to the TX fifo after a stall
+            '''
+    }
+    {
+      name: num_segment_cg
+      desc: '''
+            - Check that the DUT only transmits what is in a command even if more segments are in the TX FIFO
             '''
-    }   
+    }
+    {
+      name: segment_speed_cg
+      desc: '''
+            - Check that we test with transactions that uses different speeds
+              i.e first segment in standard mode, followed by segments in dual or quad mode
+            '''
+    }
```

Comment:
as discussed, we want to have a covergroup for passthrough. not sure if you have added it somewhere?

Reply:
@wecai good catch thank you I didn't push my last update. it is there now.
I will merge once the CI passes